CHANGELOG.md

```diff
@@ -1,3 +1,101 @@
+### 2020-09-14_1.457.0
+- Remove `account_ids` parameter from `/processor/transactions/recurring/get` request
+
+### 2020-09-14_1.456.0
+- Add `hosted_link.completion_redirect_uri` to `link/token/create` request
+
+### 2020-09-14_1.455.2
+- Deprecate max_single_transfer_amount and max_monthly_amount in /transfer/configuration/get response.
+- Deprecate monthly_transfer_volume in /transfer/metrics/get response.
+
+### 2020-09-14_1.455.1
+- Update documentation for `/credit/payroll_income/parsing_config/update`
+
+### 2020-09-14_1.455.0
+- Add new /sandbox/transfer/refund/simulate route to simulate refund events in sandbox.
+
+### 2020-09-14_1.454.0
+- Add `is_investments_fallback_item` field to `/investments/holdings/get` and `/investments/transactions/get` responses
+
+### 2020-09-14_1.453.0
+- Add `/credit/payroll_income/parsing_config/update` for updating the parsing configuration for document income verficiations
+
+### 2020-09-14_1.452.0
+- Add `warnings` field to `/cra/base_report/get` response
+
+### 2020-09-14_1.451.0
+- Add `/beacon/report_syndication/list` for listing `BeaconReportSyndication` objects linked to a specific `BeaconUser`
+
+### 2020-09-14_1.450.0
+- [Breaking] Remove `user_token` field from `asset_report/create` request.
+
+### 2020-09-14_1.449.2
+- Updates `isin` and `cusip` field descriptions for new call to action.
+
+### 2020-09-14_1.449.1
+- Add new refund event types.
+
+### 2020-09-14_1.449.0
+- Add `/beacon/report/list` for listing `BeaconReport` objects created for a specific `BeaconUser`
+
+### 2020-09-14_1.448.0
+- Make `publisher` field from `FDXNotifications` as optional
+
+### 2020-09-14_1.447.0
+- Mark `fraud_amount` as nullable for `BeaconReport` objects in `/beacon/report/create`
+
+### 2020-09-14_1.446.0
+- Add `allow_manual_entry` field to `investments` object in `link/token/create`
+- Update `/credit/freddie_mac/reports/get` return type data shape to contain one object with both VOA and VOE assets combined
+
+### 2020-09-14_1.445.0
+- Add new insights and counterparty fields for Transactions endpoints
+- Deprecate legacy category fields for Transactions endpoints
+
+### 2020-09-14_1.444.0
+- Add new webhook type `AUTHORIZATION_GRANTED` to `sandbox/item/fire_webhook`
+
+### 2020-09-14_1.443.0
+- Add `RETURN` as possible Virtual Account wallet transaction type
+
+### 2020-09-14_1.442.0
+- Add `facilitator_fee` field for the Transfers endpoints
+
+### 2020-09-14_1.441
```

Comment:
nit: I think this should be `### 2020-09-14_1.441.0` (`.0` missing) - not sure how that happened